### ISSUE TITLE
new InlineKeyboardButton types don't work when using webhook

### ISSUE THREAD

**Issue by joshuastorch** [kotlin-telegram-bot/kotlin-telegram-bot#126]
**Expected behavior**
The inlineButtons command from the [example](https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/blob/master/samples/dispatcher/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/Main.kt) should work with a [webhook setup ](https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/blob/master/samples/webhook/src/main/kotlin/com/github/kotlintelegrambot/webhook/Main.kt) the same way it does with the polling function.

**Actual behavior**
When calling the `TelegramBot.processUpdate(jsonData)` function with the webhook update data for a callback query response (a message containing reply_markup with inline_keyboard data) GSON throws a Runtime Exception while trying to deserialize com.github.kotlintelegrambot.entities.keyboard.InlineKeyboardButton()

**To Reproduce**
* Setup telegram bot with a command that sends an inline keyboard (like `command("inlineButtons")` in the  [example](https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/blob/master/samples/dispatcher/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/Main.kt) and setup a webhook instead of polling to update the bot.
* send the /inlineButtons command to the bot. The bot will reply with the inline keyboard
* when sending a response with the inline keyboard the Exception is thrown

**Example json received as webhook update (with removed IDs) that cannot be deserialized to the Update class**
````JSON
{
   "update_id":1,
   "callback_query":{
      "id":"1",
      "from":{
         "id":1,
         "is_bot":false,
         "first_name":"Joshua",
         "username":"joshua",
         "language_code":"de"
      },
      "message":{
         "message_id":1,
         "from":{
            "id":1,
            "is_bot":true,
            "first_name":"testbot",
            "username":"testbot"
         },
         "chat":{
            "id":1,
            "first_name":"Joshua",
            "username":"joshua",
            "type":"private"
         },
         "date":1606317592,
         "text":"Hello, inline buttons!",
         "reply_markup":{
            "inline_keyboard":[
               [
                  {
                     "text":"Test Inline Button",
                     "callback_data":"testButton"
                  }
               ],
               [
                  {
                     "text":"Show alert",
                     "callback_data":"showAlert"
                  }
               ]
            ]
         }
      },
      "chat_instance":"1",
      "data":"showAlert"
   }
}
````

**Screenshots/Traceback**
When trying to deserialize the json above directly with gson to com.github.kotlintelegrambot.entities.Update
````
Exception in thread "main" java.lang.RuntimeException: Failed to invoke private com.github.kotlintelegrambot.entities.keyboard.InlineKeyboardButton() with no args
	at com.google.gson.internal.ConstructorConstructor$3.construct(ConstructorConstructor.java:111)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:210)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at com.google.gson.Gson.fromJson(Gson.java:773)
	at net.jstorch.telegram.AppKt.main(App.kt:23)

````

When using the bot with webhook
````
[qtp1486553742-19] WARN io.javalin.Javalin - Uncaught exception
java.lang.RuntimeException: Failed to invoke private com.github.kotlintelegrambot.entities.keyboard.InlineKeyboardButton() with no args
	at com.google.gson.internal.ConstructorConstructor$3.construct(ConstructorConstructor.java:111)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:210)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:129)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:220)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at com.google.gson.Gson.fromJson(Gson.java:773)
	at com.github.kotlintelegrambot.UpdateMapper.jsonToUpdate(UpdateMapper.kt:8)
	at com.github.kotlintelegrambot.Bot.processUpdate(Bot.kt:192)
	at net.jstorch.telegram.TelegramBot.webHookUpdate(TelegramBot.kt:84)
	at net.jstorch.telegram.AppKt$main$1$1.handle(App.kt:76)
	at io.javalin.core.security.SecurityUtil.noopAccessManager(SecurityUtil.kt:23)
	at io.javalin.http.JavalinServlet$addHandler$protectedHandler$1.handle(JavalinServlet.kt:121)
	at io.javalin.http.JavalinServlet$service$2$1.invoke(JavalinServlet.kt:45)
	at io.javalin.http.JavalinServlet$service$2$1.invoke(JavalinServlet.kt:24)
	at io.javalin.http.JavalinServlet$service$1.invoke(JavalinServlet.kt:129)
	at io.javalin.http.JavalinServlet$service$2.invoke(JavalinServlet.kt:40)
	at io.javalin.http.JavalinServlet.service(JavalinServlet.kt:81)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at io.javalin.websocket.JavalinWsServlet.service(JavalinWsServlet.kt:51)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:763)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:569)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1610)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
	at io.javalin.core.JavalinServer$start$wsAndHttpHandler$1.doHandle(JavalinServer.kt:49)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:507)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1580)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1292)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:173)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:501)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
	at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.InstantiationException
	at java.base/jdk.internal.reflect.InstantiationExceptionConstructorAccessorImpl.newInstance(InstantiationExceptionConstructorAccessorImpl.java:48)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.google.gson.internal.ConstructorConstructor$3.construct(ConstructorConstructor.java:108)
	... 63 more
````

**Additional context**
The issue occured with Oracle JDK 11 on Windows 10 and OpenJDK 11 on Ubuntu 20.04 while using kotlin-telegram-bot 6.0.0

**Comment by vjgarciag96** [comment]
That's a really nice report! Thank you so much for that @joshuastorch! I'll take a look to it 👍

**Comment by popsa85** [comment]
Same issue. Ubuntu 20.04. Java 11 and 14

**Comment by vjgarciag96** [comment]
This should be fixed now in 6.0.1 version. Let me know if it works as expected now @joshuastorch @popsa85.

**Comment by joshuastorch** [comment]
Yes thank you, it is now working as expected!
